utils/frame/try-runtime/cli/src/commands/follow_chain.rs

```diff
@@ -223,22 +92,23 @@ where
 	ExecDispatch: NativeExecutionDispatch + 'static,
 {
 	let mut maybe_state_ext = None;
-	let (_client, subscription) = start_subscribing::<Block::Header>(&command.uri).await?;
+	let (rpc, subscription) = start_subscribing::<Block::Header>(&command.uri).await?;
 
 	let (code_key, code) = extract_code(&config.chain_spec)?;
 	let executor = build_executor::<ExecDispatch>(&shared, &config);
 	let execution = shared.execution;
 
-	let rpc_service = RpcService::new(&command.uri, command.keep_connection).await?;
```

Comment:
thus you can remove this option (`keep_connection`) from the command, since it will be always implicite a single connection